estuary_updater/handlers/freshmaker.py

```diff
@@ -111,7 +112,37 @@ def create_or_update_build(self, build, event_id):
             log.debug('Skipping build update for event {0} because it is a dry run'.format(
                 event_id))
             return
+        log.debug('Creating FreshmakerBuild {0}'.format(build['build_id']))
+        fb_params = dict(
+            id_=build['id'],
+            build_id=build['build_id'],
+            dep_on=build['dep_on'],
+            event_id=build['event_id'],
```

Comment:
This is not part of the model, since it was replaced by using a relationship with `FreshmakerEvent` nodes

Reply:
This also needs to be updated on the Estuary scraper:
https://github.com/release-engineering/estuary-api/blob/master/scrapers/freshmaker.py#L81

Reply:
This PR should address it: https://github.com/release-engineering/estuary-api/pull/237